src/code_fa4c0_len_3bf0.c

```diff
@@ -34,7 +34,12 @@ ApiStatus FadeInMusic(ScriptInstance* script, s32 isInitialCall) {
     return (func_8014A964(var0, var1, var2, var3, var4, var5) != 0) * ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "code_fa4c0_len_3bf0", func_802D5EE0);
+ApiStatus func_802D5EE0(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+
+    (&gMusicPlayers[get_variable(script, *args++)])->flags |= 0x2;
```

Comment:
does a temp for gMusicPlayers work here? Would be a bit less ugly, imo, and thus preferred

Reply:
do you mean a temp for `(&gMusicPlayers[get_variable(script, *args++)])` or just for gMusicPlayers itself?

Reply:
The latter